systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeCorsST.java

```diff
@@ -18,6 +18,7 @@
 import org.apache.kafka.clients.consumer.ConsumerConfig;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.hamcrest.CoreMatchers;
```

Comment:
Import only one method  `containsString` instead of whole class.
```suggestion
import org.hamcrest.CoreMatchers.containsString;
```